src/components/Quiz.vue

```diff
@@ -270,7 +273,8 @@ export default {
     },
     submit() {
       TrainingService.saveAnswer(this, this.picked);
-      if (TrainingService.hasCompleted(this)) {
+      // if (TrainingService.hasCompleted(this)) {
+      if (TrainingService.numAnswers === TrainingService.index || TrainingService.numAnswers > TrainingService.index) {
```

Comment:
Is this the only change related to the scoring? If so, you can commit this change directly to master.

Reply:
The bug here was caused by 

(1) the fact that `numAnswers` (the number of submitted answers) is greater than the number of questions if you re-submit an answer (i.e., if you go back and revise an answer you've already submitted, and 

(2) the equality condition blocking submission of the quiz (which is why line 273 above works around the issue...although not without introducing a new regression)

See: https://github.com/UPchieve/web/pull/160